app/components/channel_list/index.test.tsx

```diff
@@ -40,7 +44,12 @@ describe('components/channel_list', () => {
         expect(wrapper.toJSON()).toBeTruthy();
     });
 
-    it('should render team error', () => {
+    it('should render team error', async () => {
```

Comment:
I think something was broken in the channel_list header index that was preventing the header from being rendered in the failure cases, that got fixed with the changes on the way we handle permissions.

These changes are to adapt to the test now showing the header, and showing it correctly.